videotroncmdb.json

```diff
@@ -0,0 +1,302 @@
+                    "data_type": "string",
+                    "example_values": [
+                        "6000"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.CI_SupportGroup.*.Support_Group_Name",
+                    "data_type": "string",
+                    "example_values": [
+                        "Outils et actifs -> ITSP/ITSM"
+                    ]
+                },
+                {
+                    "data_path": "action_result.summary.managed_by",
+                    "data_type": "string",
+                    "example_values": [
+                        "Infrastructures -> Gestion D\u00e9couverte Serveur"
+                    ]
+                },
+                {
+                    "data_path": "action_result.summary.status",
+                    "data_type": "string",
+                    "example_values": [
+                        "Success"
+                    ]
+                },
+                {
+                    "data_path": "action_result.summary.used_by",
+                    "data_type": "string",
+                    "column_name": "Used By",
+                    "column_order": 1,
+                    "example_values": [
+                        "Outils et actifs -> ITSP/ITSM"
+                    ]
+                },
+                {
+                    "data_path": "action_result.summary.supported_by",
+                    "data_type": "string",
+                    "column_name": "Supported By",
+                    "column_order": 2,
+                    "example_values": [
+                        "Infrastructures -> Windows"
+                    ]
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string",
+                    "column_name": "Message",
+                    "column_order": 3,
+                    "example_values": [
+                        "Success"
+                    ]
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric",
+                    "example_values": [
+                        1
+                    ]
+                }
+            ],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        }
+    ],
+    "pip_dependencies": {
+        "wheel": [
+            {
+                "module": "beautifulsoup4",
+                "input_file": "wheels/beautifulsoup4-4.9.1-py2-none-any.whl"
+            },
+            {
+                "module": "requests",
```

Comment:
Please add transitive dependencies (dependencies of requests and beautifulsoup) as `pip_dependencies` and also add their wheel files. We do not need to add those dependencies in requirements.txt

Reply:
Added transitive dependencies.